### DISCUSSION TITLE
Missing powerful front-end repositories

### DISCUSSION THREAD

**Discussion by mohamadrezakoohkan** [MRezaSafari/AwesomeFrontend#19] [Ideas]
So far, I have seen things that were really interesting and enjoyable, but there is a need for other front-end repositories to develop both professional and simple projects.
For example [CWA](https://github.com/corona-warn-app) is an organization which made a lot of good work in github.
https://github.com/corona-warn-app/cwa-website

**Comment by MRezaSafari** [comment]
So! what can we do here?
is it somehow your general concern? or it's related to this repository?

**Comment by mohamadrezakoohkan** [comment]
Yes, It is related to this repo. We can provide some of them in [Full Projects](https://github.com/MRezaSafari/AwesomeFrontend/blob/main/topics/full-projects.md) section if you agree.

**Comment by MRezaSafari** [comment]
Ahaaa, I get it now.
You mean we should introduce the full open-source projects that can help people to learn more.

I agree. but I have two concerns.

first, it's not related to full projects. full projects mean the videos that teach you how to make that project.
second, mostly all project contains multiple technologies. such as html, css, react. in that case, where should we put them?
